src/applications/static-pages/simple-forms/form-upload/App.js

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback } from 'react';
+import { connect, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { isLoggedIn } from 'platform/user/selectors';
+import {
+  VaButton,
+  VaAlert,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { toggleLoginModal as toggleLoginModalAction } from '@department-of-veterans-affairs/platform-site-wide/actions';
+
+export const App = ({
+  shouldDisplayQuickSubmit,
+  toggleLoginModal,
+  formNumber,
+  hasOnlineTool,
+}) => {
+  // TODO: Confirm this accurately detects if the user is logged in.
+  const userLoggedIn = useSelector(state => isLoggedIn(state));
```

Comment:
We don't *need* to resolve this now but I want to test that this works on staging AND that the modal sign in button works. Locally, the most I've been able to test has shown different behavior on the modal button vs the log in at the top of the screen. But I think that may be as a result of me working locally rather than a deployed env.